pom.xml

```diff
@@ -120,13 +134,92 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
-        
+
+
+        <!--=======================-->
+        <!--KOTLIN-->
+        <!--=======================-->
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib-jre7</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-test-junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+
     </dependencies>
 
+    <repositories>
```

Comment:
These are not needed.
![bitmoji](https://render.bitstrips.com/v2/cpanel/9951746-288439948_4-s1-v1.png?transparent=1&palette=1&width=246)